src/main/webapp/script.js

```diff
@@ -191,6 +191,7 @@ async function getEvents(url, index, option) {
   events.forEach(function(event) {
     const eventItemElement = document.createElement('div');
     eventItemElement.className = 'event-item';
+    eventItemElement.setAttribute("onclick", "openEvent()");
```

Comment:
Use single quotes:
https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes

Reply:
Fixed, thank you!